extensions/compose/package.json

```diff
@@ -68,23 +68,23 @@
           ]
         },
         {
-          "id": "downloadComposeView",
+          "id": "downloadCommand",
           "title": "Downloading Compose ${onboardingContext:composeDownloadVersion}",
           "description": "Downloading the binary.\n\nOnce downloaded, the next step will install Compose system-wide.",
           "when": "onboardingContext:composeIsNotDownloaded == true",
-          "command": "compose.onboarding.downloadCompose",
+          "command": "compose.onboarding.downloadCommand",
           "completionEvents": [
-            "onCommand:compose.onboarding.downloadCompose"
+            "onCommand:compose.onboarding.downloadCommand"
           ]
         },
         {
-          "id": "composeFailedDownload",
+          "id": "downloadFailure",
           "title": "Failed Downloading Compose",
           "when": "onboardingContext:composeIsNotDownloaded == true",
           "state": "failed"
         },
         {
-          "id": "composeDownloaded",
+          "id": "downloadSuccessfulView",
```

Comment:
here it looks strange
it should not have a state of "completed" ?

and be only "downloadSuccessful" but not a view ?

I mean, it looks like either both of them are views or not a view

Reply:
I don't think so, because this is an intermediary step. The onboarding first downloads the binary locally, then copies it system-wide. If downloading the binary fails, this is a failure (we will retry...), but it it succeeds, other steps are following. If we were setting a `completed` state here, the onboarding would stop immediately, and it would never copy it system-wide.